software/console/Makefile

```diff
@@ -2,6 +2,7 @@ ROOT_DIR:=../..
 include $(ROOT_DIR)/system.mk
 
 SERIAL:=/dev/ttyUSB0
+LOCAL:=0
 
 CC:=gcc
```

Comment:
delete ?

Reply:
LOCAL indicated that the console was to run with the test bench.
I have now changed the name of the variable to TEST_BENCH.
So when calling "make console TEST_BENCH=1" it knows that there is no serial port and that the program will read and write to fifo files.